packages/e2e/features/ui/components/storage/storage-browser/toggle-menu.feature

```diff
@@ -0,0 +1,140 @@
+
+  @react
+  Scenario: upload, list, copy and delete a file using toggle menu
+    When I type my "email" with status "CONFIRMED"
+    Then I type my password
+    Then I click the "Sign in" button
+    When I click the first button containing "public"
+    Then I see the "Menu Toggle" button
+    When I click the "Menu Toggle" button
+    # upload file
+    Then I see the "Upload" menuitem
+    When I click the "Upload" menuitem
+    Then the "Upload" button is disabled
+    Then I upload a file with a random name
+    Then I see "Not started"
+    Then I click the label containing text "Overwrite existing files"
+    When I click the "Upload" button
+    Then I see "100%"
+    Then I see "All files uploaded"
+    When I click the "Exit" button
+    # list uploaded file
+    Then I see the file with a random name
+    # copy file
+    Then I click checkbox for random file name
+    When I click the "Menu Toggle" button
+    When I click the "Copy" menuitem
+    Then I see "Copy destination"
+    Then I click the "DoNotDeleteThisFolder_CanDeleteAllChildren/" button
+    Then I click the "Copy" button
+    Then I see "All files copied"
+    When I click the "Exit" button
+    # delete file
+    Then I click checkbox for random file name
+    When I click the "Menu Toggle" button
+    Then I click the "Delete" menuitem
+    Then I click the "Delete" button
+    Then I see "All files deleted"
+    When I click the "Exit" button
+    # delete file copy
+    When I click the first button containing "DoNotDeleteThisFolder_CanDeleteAllChildren"
+    Then I click checkbox for random file name
+    When I click the "Menu Toggle" button
+    Then I click the "Delete" menuitem
+    Then I click the "Delete" button
+    Then I see "All files deleted"
+
+  @react
+  Scenario: upload a folder
+    When I type my "email" with status "CONFIRMED"
+    Then I type my password
+    Then I click the "Sign in" button
+    When I click the first button containing "public"
+    Then I see the "Menu Toggle" button
+    When I click the "Menu Toggle" button
+    # upload file
+    Then I see the "Upload" menuitem
+    When I click the "Upload" menuitem
+    Then the "Upload" button is disabled
+    Then I upload a folder "e2eTemp" with two files with random names
+    Then I see "Not started"
+    Then I click the label containing text "Overwrite existing files"
+    When I click the "Upload" button
+    Then I see "100%"
+    Then I see "All files uploaded"
+    When I click the "Exit" button
+    # list uploaded file
+    When I click the first button containing "e2eTemp"
+    Then I see the two files with random names
+    # delete created file
+    Then I click the element with id attribute "header-checkbox"
+    When I click the "Menu Toggle" button
+    Then I click the "Delete" menuitem
+    Then I click the "Delete" button
+    Then I see "All files deleted"
+    When I click the "Exit" button
+    # verify all files are deleted
+    Then I see "No files" 
```

Comment:
Goal of UI test should be just to test the UI e2e. In this case this seems like its already covered in above test. List and delete. maybe uneccesarry unless you are covering a different UI scenarios here.

I would just check if there is a folder `e2eTemp`